services/src/handlers/datasets.rs

```diff
@@ -93,6 +93,7 @@ async fn list_external_datasets<C: Context>(
         .await?
         .list(options)
         .await?;
+    // TODO: it appears errors here lead to the internal datasets being listed because the route also matches /datasets,
```

Comment:
I guess this _must_ be fixed on the Actix change?

Reply:
in actix this shouldn't happen